<ui version="4.0" >
 <class>LayerProperties</class>
 <widget class="QWidget" name="LayerProperties" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>LayerProperties</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QTabWidget" name="_tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="_generalTab" >
      <attribute name="title" >
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="label_11" >
           <property name="text" >
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="_nameEdit" />
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Label</string>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QCheckBox" name="_labelShown" >
              <property name="text" >
               <string>Show</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="label_2" >
              <property name="text" >
               <string>Color:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QtTools::ColorButton" name="_labelColor" >
              <property name="text" >
               <string>PushButton</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="label_3" >
              <property name="text" >
               <string>Column:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="_labelColumn" />
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="label_4" >
              <property name="text" >
               <string>Size:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="_labelSize" >
              <property name="alignment" >
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="maximum" >
               <number>10000</number>
              </property>
              <property name="singleStep" >
               <number>5</number>
              </property>
              <property name="value" >
               <number>25</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="label_5" >
              <property name="text" >
               <string>Z Offset:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="_labelZOffset" >
              <property name="alignment" >
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="maximum" >
               <number>10000000</number>
              </property>
              <property name="singleStep" >
               <number>50</number>
              </property>
              <property name="value" >
               <number>1000</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>Legend</string>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QCheckBox" name="_showInLegend" >
              <property name="text" >
               <string>Show</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="label_6" >
              <property name="text" >
               <string>Text:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="_legendText" />
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QCheckBox" name="_showCountLegend" >
              <property name="text" >
               <string>Show Count</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QCheckBox" name="_showMinLegend" >
              <property name="text" >
               <string>Show Minimium</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QCheckBox" name="_showMaxLegend" >
              <property name="text" >
               <string>Show Maximium</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5" >
         <property name="title" >
          <string>Offset</string>
         </property>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="label_12" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>X:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="_xOffset" />
          </item>
          <item>
           <widget class="QLabel" name="label_13" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Y:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="_yOffset" />
          </item>
          <item>
           <widget class="QLabel" name="label_14" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Z:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="_zOffset" />
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="_drawingTab" >
      <attribute name="title" >
       <string>Drawing</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="title" >
          <string>Color</string>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="label" >
              <property name="text" >
               <string>Coloring Method:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="_colorTypeComboBox" />
            </item>
           </layout>
          </item>
          <item>
           <widget class="QStackedWidget" name="_colorStack" >
            <widget class="QWidget" name="page_3" />
            <widget class="QWidget" name="page_4" />
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4" >
         <property name="title" >
          <string>Primitive</string>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QStackedWidget" name="_primitiveStack" >
            <widget class="QWidget" name="page_5" />
            <widget class="QWidget" name="page_6" />
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="_timeTab" >
      <attribute name="title" >
       <string>Time</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="label_9" >
           <property name="text" >
            <string>Minimium Date:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_2" />
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit" />
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="label_10" >
           <property name="text" >
            <string>Maximium Date:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox" />
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit_2" />
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="_queryTab" >
      <attribute name="title" >
       <string>Query</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QCheckBox" name="_customQuery" >
         <property name="text" >
          <string>Custom Query</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="label_8" >
           <property name="text" >
            <string>Primary Key:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="_primaryKeyText" />
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_7" >
         <property name="text" >
          <string>Query:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="_queryEdit" >
         <property name="acceptRichText" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtTools::ColorButton</class>
   <extends>QPushButton</extends>
   <header>QtTools/ColorButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
